docs/streams/upgrade-guide.html

```diff
@@ -72,11 +72,16 @@ <h1>Upgrade Guide and API Changes</h1>
         More details about the new config <code>StreamsConfig#TOPOLOGY_OPTIMIZATION</code> can be found in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-295%3A+Add+Streams+Configuration+Allowing+for+Optional+Topology+Optimization">KIP-295</a>.
     </p>
 
-    <h3><a id="streams_api_changes_250" href="#streams_api_changes_250">Streams API changes in 2.5.0</a></h3>
-    <p>
-        As of 2.5.0 Kafka we deprecated <code>UsePreviousTimeOnInvalidTimestamp</code> and replaced it with <code>UsePartitionTimeOnInvalidTimeStamp</code> as per
-        <a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=130028807">KIP-530</a>
-    </p>
+    <h3><a id="streams_api_changes_250" href="#streams_api_changes_250">Streams API changes in 2.5.0</a>
+        <p>
+            We have added <code>CogroupedKStream</code>, which can be used to aggregate <code>KGroupStreams</code> into a <code>KTable</code>.
+            <code>TimeWindowedCogroupedKStream</code> and <code>SessionWindowedCogroupedKStream</code> have also been added to provide windowing support to cogroups.
+            <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html"></a>
+        </p>
+        <p>
+            As of 2.5.0 Kafka we deprecated <code>UsePreviousTimeOnInvalidTimestamp</code> and replaced it with <code>UsePartitionTimeOnInvalidTimeStamp</code> as per
+            <a href="https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=130028807">KIP-530</a>
+        </p>
```

Comment:
Why did you change the indention? This implies an unnecessary diff as the last 4 lines did not change. (For future reference -- it makes reviewing easier (I was unnecessarily checking what you have changed here...)

Reply:
ah, again the merge didn't work the way I would expect